pkg/kusionctl/cmd/preview/options.go

```diff
@@ -1,25 +1,41 @@
 package preview
 
 import (
-	applycmd "kusionstack.io/kusion/pkg/kusionctl/cmd/apply"
+	"fmt"
+	"os"
+
+	"github.com/pterm/pterm"
+
+	"kusionstack.io/kusion/pkg/compile"
+	"kusionstack.io/kusion/pkg/engine/backend"
+	"kusionstack.io/kusion/pkg/engine/models"
+	"kusionstack.io/kusion/pkg/engine/operation"
+	opsmodels "kusionstack.io/kusion/pkg/engine/operation/models"
+	"kusionstack.io/kusion/pkg/engine/operation/types"
+	"kusionstack.io/kusion/pkg/engine/runtime"
+	runtimeInit "kusionstack.io/kusion/pkg/engine/runtime/init"
+	"kusionstack.io/kusion/pkg/engine/states"
 	compilecmd "kusionstack.io/kusion/pkg/kusionctl/cmd/compile"
+	"kusionstack.io/kusion/pkg/log"
+	"kusionstack.io/kusion/pkg/projectstack"
+	"kusionstack.io/kusion/pkg/status"
 )
 
 type PreviewOptions struct {
 	compilecmd.CompileOptions
-	Yes     bool
-	Detail  bool
-	NoStyle bool
+	PreviewFlags
+	backend.BackendOps
+}
+
+type PreviewFlags struct {
+	Operator     string
+	Detail       bool
+	IgnoreFields []string
```

Comment:
add usages in our doc for this feature -ignore-fields

Reply:
I added usage in the cmd example